src/main/java/bio/terra/cloudfunctions/common/StorageObjectEventHarness.java

```diff
@@ -29,7 +29,7 @@
    */
   @Override
   public void accept(String json, Context context) throws Exception {
-    message = GsonWrapper.getInstance().fromJson(json, StorageObjectData.class);
+    message = GsonWrapper.convertFromClass(json, StorageObjectData.class);
```

Comment:
I suggest wrapping this in a try/catch, and logging any json-parsing errors. This will also make use of the `logger`, which is otherwise unused!

Reply:
Included try/catch block to catch Gson exceptions.